validator/src/main/java/com/amazon/aoc/validators/ConatinerInsightECSStructuredLogValidator.java

```diff
@@ -0,0 +1,69 @@
+package com.amazon.aoc.validators;
```

Comment:
maybe add a unittest for it?

Reply:
Can I block Ying for my future PR, I didn't write any unit test for ecs prometheus test /w